src/Microsoft.Tye.Hosting/DockerImagePuller.cs

```diff
@@ -66,16 +64,39 @@ private async Task PullContainerAsync(string image)
         {
             await Task.Yield();
 
+            string name = image;
+            string version = "latest";
+            var idx = image.LastIndexOf(":");
+            if (image.IndexOf("/") < idx)
+            {
+                name = image.Substring(0, idx);
+                version = image.Substring(idx + 1);
+            }
+
+            bool installed = false;
+            string output = string.Empty;
+            await ProcessUtil.RunAsync(
```

Comment:
What does this return? What happens if the exit code is non zero?

Reply:
Throws CommandException